components/shared/webrender/Cargo.toml

```
@@ -27,3 +27,4 @@ serde = { workspace = true }
 servo_geometry = { path = "../../geometry" }
 servo-media = { workspace = true }
 surfman = { workspace = true, features = ["sm-x11"] }
+style = { workspace = true }
```

I suggest keeping these in alphabetical order. It's surprising that this wasn't caught by the lint.

```suggestion
style = { workspace = true }
surfman = { workspace = true, features = ["sm-x11"] }
```